test/systemtest/functional_fpga_test.py

```diff
@@ -53,13 +53,22 @@ def spiflash_proc(self, tmp_path, spiflash, sw_test_bin):
 
         p_flash.terminate()
 
-    def test_execute_binary(self, spiflash_proc, fpga_uart, uart_timeout):
+    def test_execute_binary(self, spiflash_proc, fpga_uart, uart_timeout, logfile):
         """
         Executes the binary and inspects its UART for "PASS!\r\n" or "FAIL!\r\n".
         """
 
         logger = logging.getLogger(__name__)
 
+        if logfile:
+            logger.debug("Logfile at" + logfile)
```

Comment:
You're missing a space, write `"Logfile at " + logfile` (or, for extra clarity, `"Logfile at %s" % (logfile,)`, but that one's kind of overkill).